doc/languages-frameworks/rust.section.md

```diff
@@ -20,12 +20,16 @@ or use Mozilla's [Rust nightlies overlay](#using-the-rust-nightlies-overlay).
 Rust applications are packaged by using the `buildRustPackage` helper from `rustPlatform`:
 
 ```nix
-{ lib, rustPlatform }:
+{ lib, fetchFromGitHub, rustPlatform }:
 
-rustPlatform.buildRustPackage rec {
```

Comment:
`rec` on the builder is cleaner than `let`ting the `pname` and `version`, however.

Reply:
Ah yeah. I'm still leaning a bit ore towards the let side in this discussion, but I agree, rec is not an anti-pattern in this case. Will drop the second commit.